### ISSUE TITLE
Sign in links fail WCAG AA standards when hovered (red)

### ISSUE THREAD

**Issue by FestiveKyle** [canada-ca/tracker#2462]
Sign in links in header have a 3.54 contract when hovered (using the red #E65225), AA requires at least 4.5.

![image](https://user-images.githubusercontent.com/47400288/122449808-a9682080-cf7c-11eb-88e9-88a11387a551.png)
